common/level.go

```diff
@@ -5,44 +5,146 @@ import (
 	"engo.io/gl"
 )
 
+// Level is a parsed TMX level containing all layers and default Tiled attributes
 type Level struct {
-	width      int
-	height     int
-	TileWidth  int
+	// Orientation is the parsed level orientation from the TMX XML, like orthogonal
```

Comment:
Suggestion: `TMX XML` to `TMX file` and add in another example to `like orthogonal ....`